src/mumble/AudioOutputUser.cpp

```diff
@@ -5,11 +5,12 @@
 
 #include "AudioOutputUser.h"
 
-AudioOutputUser::AudioOutputUser(const QString& name) : qsName(name) {
-	iBufferSize = 0;
-	pfBuffer = nullptr;
-	pfVolume = nullptr;
-	fPos[0]=fPos[1]=fPos[2]=0.0;
+AudioOutputUser::AudioOutputUser(const QString& name, unsigned int bufferSize) : qsName(name) {
+	iBufferSize = bufferSize;
+	if (bufferSize > 0) {
```

Comment:
If the buffer size is `0`, then it doesn't make a whole lot sense to create this object, does it?

I'd remove the `if` and insert a `Q_ASSERT(bufferSize > 0)` instead.

Reply:
Ehhhh... Actually, if the buffer size is 0, it means that the buffer will not be initialized by the constructor, but being construct by the problematic line mentioned above. In the past, all buffers are allocated by that line, but this time I provide a way to initialize the buffer when constructing this object. Now I only made CoreAudio and PulseAudio work this way, therefore other audio backends still rely on the latter method to initialize the buffer.

Reply:
Ah okay - then I guess it depends on whether we're going to aim for removing the `resizeBuffer` function or not...